Firmware/Marlin_main.cpp

```diff
@@ -1247,7 +1247,7 @@ void setup()
     temp_mgr_init();
 
 #ifdef EXTRUDER_ALTFAN_DETECT
-	SERIAL_ECHORPGM(_n("Extruder fan type: "));
+	SERIAL_ECHORPGM(_n("Hotend fan type: "));
```

Comment:
I wouldn't touch this one. It's not visible anyway and prusa link might be parsing it

Reply:
As "Hotend fan" is the only term we should use from now on I will create an issue for PrusaLink and add a Release Note. There might be some OctoPrint Plugins that depend on it.

Reply:
@leptun Thanks pointing this out and I have found something related in PrusaLink.